README.md

```diff
@@ -57,8 +57,11 @@ window.demoLinker(
             url: 'javascript/#collapse'
         }
     },
-    /* options */ {
-        baseUrl: 'https://getbootstrap.com/'
+    /* options */
+    {
+        baseUrl: 'https://getbootstrap.com/',
+    /* add rainbow colored outlines https://gist.github.com/addyosmani/fd3999ea7fce242756b1 */
```

Comment:
I'd move this comment to the code (as a reference to where we got it from) rather than making it part of the readme.